### ISSUE TITLE
Multi-server clusterting support

### ISSUE THREAD

**Issue by DanWin** [DanWin/hosting#78]
Allow distribution among multiple servers for better scalability and reliability. This should include mysql clustering to share relevant updates across multiple servers, assigning a user account to a specific server (or multiples for load balancing), moving accounts between servers and a fair distribution of new accounts.
File systems could stay separate on each server, but using a distributed network file system would be optimal for fast account moving and load balancing on several servers for high-traffic services.

**Comment by linhux** [comment]
Yes like cloudmin, there you can move domains from server to server with all privileges.
